plasma_framework/contracts/src/exits/payment/controllers/PaymentStartInFlightExit.sol

```diff
@@ -309,14 +284,50 @@ library PaymentStartInFlightExit {
 
     function setInFlightExitInputs(
         PaymentExitDataModel.InFlightExit storage ife,
-        PaymentTransactionModel.Transaction[] memory inputTxs,
-        UtxoPosLib.UtxoPos[] memory inputUtxosPos
+        StartExitData memory exitData
     )
         private
     {
-        for (uint i = 0; i < inputTxs.length; i++) {
-            uint16 outputIndex = inputUtxosPos[i].outputIndex();
-            ife.inputs[i] = inputTxs[i].outputs[outputIndex];
+        for (uint i = 0; i < exitData.inputTxs.length; i++) {
+            uint16 outputIndex = exitData.inputUtxosPos[i].outputIndex();
+            WireTransaction.Output memory output = WireTransaction.getOutput(exitData.inputTxs[i], outputIndex);
+
+            address payable exitTarget;
+            if (exitData.inputUtxosTypes[i] == 0) {
```

Comment:
You'll need to change this when merging with master. As I stated in another comment we changed the way of handling outputGuard

Reply:
done